winpr/libwinpr/file/generic.c

```diff
@@ -461,6 +468,112 @@ BOOL FlushFileBuffers(HANDLE hFile)
 	return FALSE;
 }
 
+BOOL WINAPI GetFileAttributesExA(LPCSTR lpFileName, GET_FILEEX_INFO_LEVELS fInfoLevelId,
+																 LPVOID lpFileInformation)
+{
+	LPWIN32_FILE_ATTRIBUTE_DATA fd = lpFileInformation;
+	WIN32_FIND_DATAA findFileData;
+	HANDLE hFind;
+
+	if (!(hFind = FindFirstFileA(lpFileName, &findFileData) == INVALID_HANDLE_VALUE))
```

Comment:
@DavBfr Not really, only if the inversion of hFind equals ```INVALID_HANDLE_VALUE```. That really depends on the definition. It also produces compiler warnings with C11